tests/benchmark/benchmark_between_releases/Dockerfile.in

```diff
@@ -0,0 +1,24 @@
+FROM ${FULL_BASE_IMAGE}
+
+# Install performance
+RUN pip install performance
+
+# Create virtual environment
+RUN pip install --upgrade virtualenv
+
+# Download ensurepip module which prevents the failure when benchmarking python3, can be removed once we drop debian's 3.4
+RUN wget https://www.python.org/ftp/python/3.4.2/Python-3.4.2.tgz
+RUN tar xzf Python-3.4.2.tgz
+RUN cp -R Python-3.4.2/Lib/ensurepip /usr/lib/python3.4
+
+RUN mkdir /${TAG}
```

Comment:
You should be able to store the results in /app in each Docker image, then copy them to different directories on the host.